docs/en/ingest-management/fleet/fleet-server-secrets.asciidoc

```diff
@@ -0,0 +1,252 @@
+[[fleet-server-secrets]]
+= {fleet-server} Secrets
+
+{fleet-server} configuration will contain secret values.
+These values may be specified directly in the configuration or through secret files.
+Command line args may be used to pass the values or file paths when running under the {agent}, or environment variables may be used if the {agent} is running in a container.
+
+NOTE: Stand-alone {fleet-server} is under active development.
+
+== Secret Values
```

Comment:
```suggestion
For examples of how to deploy secret files, refer to our <<secret-files-guide,Secret files guide>>.

NOTE: Stand-alone {fleet-server} is under active development.

[discrete]
== Secret Values
```
This `[discrete]` tag will keep the "Secret values" section on the same page as the top section. The "Secret values Guide" works nicely as a separate page, so we could add a link right here to make sure people find the examples.